src/api/v1/employees/routes/create.ts

```diff
@@ -0,0 +1,45 @@
+import 'reflect-metadata'
+import { FastifyReply, FastifyRequest } from 'fastify'
+import * as jf from 'joiful'
+
+interface Body {
+  firstName: string
+  lastName: string
+  title: string
+  department: string
+  salary: number
+  secLevel: number
+  permanent: boolean
+  projects: string[]
+}
```

Comment:
Consistent spacing pls
```suggestion
}

```